SU-HSDO/suhumsci
# Fix: Video w/ Caption JS

# READY FOR REVIEW

## Summary
Adds a missing bracket in `media/video-with-caption.js`

## Need Review By (Date)
asap

## Urgency
high
## Steps to Test
1. Run `npm test`
2. Check that the JS runs on a page with a video with caption
3. Check that there are no errors in the console

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
